grafast/grafserv/__tests__/exampleServer.ts

```diff
@@ -36,8 +36,7 @@ export async function makeExampleServer(
   });
 
   const serv = grafserv({ schema, preset });
-  const server = createServer();
-  serv.addTo(server);
+  const server = createServer(serv.createHandler());
```

Comment:
Please don't make this change, it does not handle websockets and other concerns. It means that `serv` is not passed an instance of `server` and thus cannot add any event listeners.

Reply:
I'm not sure this would work, wouldn't that make the WHATWG adapter reliant on node types?
I also see that the lambda adapter does not have an addTo implementation.

For now I changed the `makeExampleServer` function to receive a type parameter to revert the node behavior to addTo while keeping the different behavior for whatwg. Let me know if that works.